neo/Prompt/CommandBase.py

```diff
@@ -0,0 +1,130 @@
+from abc import ABC, abstractmethod
+from neo.Prompt.Utils import get_arg
+from typing import List
+
+
+class ParameterDesc():
+    def __init__(self, name, description, optional=False):
+        self.name = name
+        self.description = description
+        self.optional = optional
+
+    def __repr__(self):
+        if self.optional:
+            return f"{self.name:<15} - (Optional) {self.description}"
+        else:
+            return f"{self.name:<15} - {self.description}"
+
+    def formatted_name(self):
+        if self.optional:
+            return f"({self.name})"
+        else:
+            return f"{{{self.name}}}"
+
+
+class CommandDesc():
+    def __init__(self, command, short_help, help, params: List[ParameterDesc] = None):
+        """
+        Command descriptor
+
+        Args:
+            command: 1 word command identifier
+            short_help: short description of the purpose of the command
+            help: '???'
```

Comment:
remove `help` from docstring, from function parameter and other parts. I think we concluded it's not used now. I think at this unused stage it will only lead to confusion what to enter there, so it's better to remove